### PR TITLE
remove sass as a dependency

### PR DESCRIPTION
**Pull request by pYr0x**
make the modlet compatible to the canjs component.
refactor component-modlet #5 
@Alfredo-Delgado @moschel @DesignByOnyx what do you think?

### CODE REVIEW

**Comment by pYr0x** [comment]
Going to merge this tomorrow.
Pls disagree if wanted @Alfredo-Delgado

File: package.json [hunk 1]
```diff
@@ -1,11 +1,8 @@
 {
   "name": "donejs-react",
-  "version": "0.0.5",
+  "version": "0.1.0",
```
Comment (Alfredo-Delgado) [inline package.json]: Works for me 👍

**Comment by pYr0x** [comment]
@Alfredo-Delgado can you push this release to npm pls... thanks

**Comment by Alfredo-Delgado** [comment]
@pYr0x Done: https://www.npmjs.com/package/donejs-react

**Comment by pYr0x** [comment]
Thx!

**Pull request merged**